src/pages/Payroll/Payroll.tsx

```diff
@@ -109,8 +204,36 @@ function formatDate(
           href: "",
         },
       ]}
-      actions={<Button variant="contained" endIcon={<CaretRight />}>Start a payroll</Button>}
+      actions={
+        <Button
+          variant="contained"
+          onClick={() =>
+            navigate("/payroll/preview", {
+              state: { uniqueFarmId, uniqueSeasonId , uniqueSeasonNa},
```

Comment:
pass the parameters, with query params, and don't pass the season name, there should be a request to get the season information with the id on the preview

Reply:
The endpoint on the preview page does not return the name, and we need it for the modal

Reply:
P2-280-add-season-name-on-preview-endpoint-be